proofs/README.md

```diff
@@ -0,0 +1,51 @@
+# `turnkey_proofs`
+
+This crate contains utilities to parse and verify Turnkey enclave proofs. As outlined in [the Turnkey whitepaper](https://whitepaper.turnkey.com) there are two types of proofs:
+* App proofs, signing structured data with enclave ephemeral keys.
+* Boot proofs, which are proofs that a given enclave was provisioned correctly. Boot proofs reference, among other things, enclave ephemeral key. 
```

Comment:
worth adding a little bit of additional color on what `reference` means in this case?

Reply:
yep, good call.

Reply:
Addressed in https://github.com/tkhq/rust-sdk/pull/18/commits/5ecfbef882ef89ec5c77b2c2c18b7f820dfaf59c